tools/db_ping_remote.rb

```diff
@@ -1,10 +1,10 @@
+require File.expand_path('../config/environment', __dir__)
```

Comment:
> I assume there is a reason they are not already, but it would just cut down on the typing further:

Probably because of appliance rpm business.  I believe shebang lines will work properly in the appliance multiple ruby environment with system ruby from CentOS/RHEL and having packaged 2.2+ ruby from software collections. I don't have an appliance with system ruby rpm and scl ruby rpm at the same time.